test/apps/network/udp_err.c

```diff
@@ -154,6 +154,32 @@ FN_TEST(bind)
 }
 END_TEST()
 
+FN_TEST(bind_reuseaddr)
+{
+	sk_addr.sin_port = htons(8081);
+	struct sockaddr *psaddr = (struct sockaddr *)&sk_addr;
+	socklen_t addrlen = sizeof(sk_addr);
+
+	int enable = 1;
+	int socket1 = CHECK(socket(PF_INET, SOCK_DGRAM | SOCK_NONBLOCK, 0));
+	int socket2 = CHECK(socket(PF_INET, SOCK_DGRAM | SOCK_NONBLOCK, 0));
```

Comment:
`socket1`/`socket2` -> `sk1`/`sk2`, just for consistency with `sk_unbound`, `sk_bound`, e.t.c.